bolt-design-system/bolt
# <bolt-button> Component Testing Checklist

## Jira
Related to #951

## Summary
Defines the initial `<bolt-button>` component testing checklist for both automated and manual testing

## How to test
Read through initial docs to confirm each item makes sense and can be verified is working as expected with the detail provided.

> Note: Additional work creating supporting examples of these to simplify manual testing steps + fleshing out additional details for non-tech reviewers can be part of a part 2 PR we create -- this is meant to be a quick overview of the current state of things so we can keep iterating on this.
